modules/admin_manual/pages/release_notes.adoc

```diff
@@ -27,6 +28,90 @@
 * xref:changes-in-8-0[Changes in 8.0]
 * xref:changes-in-7-0[Changes in 7.0]
 
+== Changes in 10.4.0
+
+Dear ownCloud administrator, please find below the changes and known issues in ownCloud Server 10.4 that need your attention.
+You can also read {owncloud-changelog-url}[the full ownCloud Server changelog] for further details on what has changed.
+
+=== Migrations
+
+Upgrading from ownCloud Server 10.3.x to 10.4.0 does not involve database migrations. 
+The upgrade duration is, therefore, expected to be short.
+
+=== PHP 7.0 Support Discontinued
+
+As xref:release_notes.html#php-7-0-deprecation-note[announced], in the previous release of ownCloud Server, from version 10.4 onward, ownCloud **no longer supports PHP 7.0**. 
+If you're running on PHP 7.0, it is necessary to upgrade PHP **prior** to conducting the upgrade to Server 10.4. 
+We strongly recommend upgrading to PHP >= 7.2. 
```

Comment:
```suggestion
We strongly recommend upgrading to PHP 7.2 or 7.3. 
```
Now that PHP 7,4 is out, we had better specify 7.2/7.3. With ownCloud 10.4 we do not have PHP 7.4 support.